code/clean_names.R

```diff
@@ -0,0 +1,128 @@
+library(tidyverse)
+library(RPostgreSQL)
+library(httr)
+library(fuzzyjoin)
+library(refinr)
+library(stringdist)
+
+# thought about making nickname replacements (rob --> robert) but it seems like some of those are actually different people...only make replacements if IDs are the same
+# e.g. str_replace_all(name, c(!!!nicknames))
+# nicknames <- tribble(
+#   ~short,     ~long,
+#   "ROB",      "ROBERT",
+#   "DAVE",     "DAVID"
+# ) %>%
+#   mutate(short = sprintf("\\b%s\\b", short)) %>%
+#   deframe()
+abbr <- tribble(
```

Comment:
This is rad. Something I noticed was a ton of abbreviations also for rank or designation, e.g., "LT" for lieutenant, "Det" for detective